new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItConfigDistributionStrategy.java

```diff
@@ -856,10 +856,12 @@ private void createDomain() {
             .serverPod(new ServerPod() //serverpod
                 .addEnvItem(new V1EnvVar()
                     .name("JAVA_OPTIONS")
-                    .value("-Dweblogic.debug.DebugSituationalConfig=true"))
-                .addEnvItem(new V1EnvVar()
-                    .name("JAVA_OPTIONS")
-                    .value("-Dweblogic.debug.DebugSituationalConfigDumpXml=true"))
+                    .value("-Dweblogic.debug.DebugSituationalConfig=true "
+                        + "-Dweblogic.debug.DebugSituationalConfigDumpXml=true "
+                        + "-Dweblogic.kernel.debug=true "
+                        + "-Dweblogic.debug.DebugMessaging=true "
+                        + "-Dweblogic.debug.DebugConnection=true "
+                        + "-Dweblogic.ResolveDNSName=true"))
```

Comment:
do we need all of these debug flags?

Reply:
At least for now we need these flags. These are tests with intermittent failures. In case test fails we will have the logs ready to debug.